fedcloud_monitoring_tools/vo_mon.py

```diff
@@ -0,0 +1,157 @@
+"""VO-level testing for a given site in a VO"""
+
+import io
+import os
+import time
+from datetime import datetime, timezone
+
+import click
+import paramiko
+from fabric import Connection
+from imclient import IMClient
+
+IM_REST_API = "https://im.egi.eu/im"
+AUTH_FILE = "auth.dat"
```

Comment:
is this meant to persist between executions? It looks like just a temp file, why not using [tempfile](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile)?

Reply:
it is helpful to keep it in a known location and when unexpected things happen I can easily troubleshoot directly with the `im_client.py` CLI tool.

Reply:
ok